### PR TITLE
adding some missing targets into rclone.conf

### PR DESCRIPTION
**Pull request by Kelvinrr**
## Description

rlcone.conf had some module missing for some reason, this adds them back in. 

## Related Issue

https://github.com/USGS-Astrogeology/ISIS3/issues/5038

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/scripts/downloadIsisData [hunk 1]
```diff
@@ -159,35 +157,51 @@ def main(mission, dest, cfg_path, dry_run, ntransfers):
...
+
+    This script combines multiple sources using the rclone command under the hood. 
+    
+    After positional args, accepts arbitrary rclone flags. Run "rclone help flags" to get a full list of rclone flags. 
+    
+    See the ISIS repo (https://github.com/USGS-Astrogeology/ISIS3) for more info. 
+    
+    Examples of how to run this program:
+        
+        # download everything for tgo
```
Comment (jlaura) [inline isis/scripts/downloadIsisData:188]: download everything for the tgo cassis mission

File: isis/scripts/downloadIsisData [hunk 2]
```diff
@@ -159,35 +157,51 @@ def main(mission, dest, cfg_path, dry_run, ntransfers):
...
+        $ downloadIsisData tgo $ISISDATA
+    
+        # download everythin except kernels with debug output
```
Comment (jlaura) [inline isis/scripts/downloadIsisData]: everything

File: isis/scripts/downloadIsisData [hunk 3]
```diff
@@ -159,35 +157,51 @@ def main(mission, dest, cfg_path, dry_run, ntransfers):
...
+        $ downloadIsisData all $ISISDATA --dry-run --exclude="kernels/**" -vv
+        
+        # download only cks and fks
+        $ downloadIsisData all $ISISDATA --include="{ck/**,fk/**}"''')
```
Comment (jlaura) [inline isis/scripts/downloadIsisData]: On the off change someone copies this line from the source, maybe push the `''')` to a newline?

File: isis/scripts/downloadIsisData [hunk 4]
```diff
@@ -159,35 +157,51 @@ def main(mission, dest, cfg_path, dry_run, ntransfers):
...
+    parser.add_argument('mission', help='mission for files to be downloaded. Use "all" to download everything, "legacybase" to download base data for ISIS<4.1.0.')
     parser.add_argument('dest', help='the destination to download files from source')
-    parser.add_argument('--dry-run', help="run a dry run for rclone value should be a boolean", default=False, action='store_true')
     parser.add_argument('-v', '--verbose', action='count', default=0)
     parser.add_argument('-n', '--num-transfers', action='store', default=10)
     parser.add_argument('--config', action='store', default=find_conf())
-    args = parser.parse_args()
+    args, kwargs = parser.parse_known_args()
+    
+    print(kwargs)
```
Comment (jlaura) [inline isis/scripts/downloadIsisData]: Do you want this print in here? Maybe a logger with debug is better?

File: isis/scripts/downloadIsisData [hunk 5]
```diff
@@ -159,35 +157,51 @@ def main(mission, dest, cfg_path, dry_run, ntransfers):
...
-                args = create_rclone_arguments(dest, remote, dry_run, ntransfers)
-                rclone(command="sync", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
+                args = create_rclone_arguments(dest, remote, ntransfers, kwargs)
+                rclone(command="copy", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
     else:
         for remote in supported_missions[mission]:
-            args = create_rclone_arguments(dest, remote, dry_run, ntransfers)
-            rclone(command="sync", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
+            args = create_rclone_arguments(dest, remote,  ntransfers, kwargs)
+            rclone(command="copy", extra_args=args, config=cfg_path, redirect_stdout=False, redirect_stderr=False)
```
Comment (jlaura) [inline isis/scripts/downloadIsisData:171]: This swap will cause existing files to be re-downloaded? In essence, is the rclone sync vs. copy the same different from the aws s3 sync vs. copy?

File: isis/scripts/downloadIsisData (hunk 1 above)
Comment (jessemapel) [reply isis/scripts/downloadIsisData:188]: Just TGO for the mission name. CaSSIS is the sensor.

File: isis/scripts/downloadIsisData (hunk 5 above)
Comment (Kelvinrr) [reply isis/scripts/downloadIsisData:171]: `copy` checks if the file exists before re-copying. It should only re-copy if 1. the same file exists on different sources and 2. the file is newer. I would say re-download if another source has a newer version of the same file is probably desired? Shouldn't happen too much after we sanitize the USGS sources.

File: isis/scripts/downloadIsisData (hunk 4 above)
Comment (Kelvinrr) [reply isis/scripts/downloadIsisData]: This was temporary so I easily look at them without them being instantly swallowed in the pile of debug prints when -vv is passed. Should have been removed.

File: isis/scripts/downloadIsisData (hunk 5 above)
Comment (jessemapel) [reply isis/scripts/downloadIsisData:171]: Agreed, we want to take newer files, but it shouldn't happen too often. The kernels are always versioned as separate files

**Review by jessemapel — APPROVED** [review]

**Pull request merged**